page_creation.rst

```diff
@@ -133,9 +133,9 @@ variable so we can render that::
         {
             $number = mt_rand(0, 100);
 
-            return $this->render('lucky/number.html.twig', array(
-                'number' => $number
-            ));
+            return $this->render('lucky/number.html.twig', [
+                'number' => $number,
+            ]);
```

Comment:
@xabbuh Ah, it makes sense. So making changes to the new array syntax is pointless in docs? I'm just wondering for myself for the future.